src/hooks/routing/useRouterTrade.ts

```diff
@@ -94,40 +80,26 @@ export function useRouterTrade<TTradeType extends TradeType>(
     }
   }, [route, tradeType])
 
+  // get USD gas cost of trade in active chains stablecoin amount
+  const gasUseEstimateUSD = useStablecoinAmountFromFiatValue(data?.gasUseEstimateUSD)
+
   return useMemo(() => {
     if (!currencyIn || !currencyOut) return INVALID_TRADE
 
     if (!trade && !isError) {
-      return { state: isDebouncing ? TradeState.SYNCING : TradeState.LOADING, trade: undefined }
+      return { state: TradeState.LOADING, trade: undefined }
```

Comment:
should we extract this return value and the one for `TradeState.NO_ROUTE_FOUND` to the global scope to keep refs stable, similar to `INVALID_TRADE`?